### PR TITLE
Update ptf-bucket to include 1.17 dummy

### PR DESCRIPTION
**Pull request by Roavill2**
Please check if your PR fulfills the following requirements. This is simply a reminder of what we are going to look for before merging your PR. If you don't know all of this information when you create this PR, don't worry. You can edit this template as you're working on it.

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Necessary documentation (if appropriate) have been added / updated
- [x] DCO signoffs have been added to all commits, including this PR

#### PR type
What type of changes does your PR introduce to Zowe? _Put an `x` in the box that applies to this PR. If you're unsure about any of them, don't hesitate to ask._ 
- [ ] Bugfix 
- [ ] Feature 
- [ ] Other... Please describe:

#### Relevant issues

Fixes 

#### Changes proposed in this PR

- Update the APAR and PTF numbers in the PTF bucket file

#### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

#### Does this PR do something the person installing Zowe should know about?

---
* Affected function: _general area of interest_ *

---
* Description: _1 line description_ *

---
* Part: _name of customizable file involved_  *

---
_multi-line description_

#### Is there a related doc issue or Pull Request? 

Doc issue/PR number:

### CODE REVIEW

**Comment by stevenhorsman** [comment]
What is the reason for this change? Are they the correct PTF and APAR numbers?

File: smpe/bld/service/ptf-bucket.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@
 #UO01942 UO01943 - IO27757 IO27758 - Sat May  2 01:01:51 UTC 2020 - Zowe 1.11.0
 #UO01945 UO01946 - IO27798 IO27799 - Fri Jun 12 15:37:27 UTC 2020 - Zowe 1.12.0
 #UO01948 UO01949 - IO27918 IO27919 - Fri Jul 10 13:23:34 UTC 2020 - Zowe 1.13.0
-#UO01951 UO01952 - IO27949 IO27950 - Wed Aug 12 14:01:06 UTC 2020
-#UO01953 UO01954 - IO27951 IO27952 - Fri Sep 11 12:48:24 UTC 2020
-UO01955 UO01956 - IO27953 IO27954
-
+#UO01951 UO01952 - IO27949 IO27950 - Wed Aug 12 14:01:06 UTC 2020 - Zowe 1.14.0
+#UO01953 UO01954 - IO27951 IO27952 - Fri Sep 11 12:48:24 UTC 2020 - Zowe 1.15.0
+#UO01955 UO01956 - IO27953 IO27954 -
```
Comment (stevenhorsman) [inline smpe/bld/service/ptf-bucket.txt:8]: Commenting out the current APAR number will break the build

**Review by stevenhorsman — CHANGES_REQUESTED** [review]

**Comment by Roavill2** [comment]
I was given these APAR and PTF numbers by Nayer at the end of last week:

"I just wanted to let you know that dummy APAR/PTF number for Zowe 1.17 is created in RETAIN/zService:
IO27974 - UO01958
IO27975 - UO01959
So that you could update ptf-bucket.txt."

I've undone the change to the current APAR number but made sure the new ones are commented out.

File: smpe/bld/service/ptf-bucket.txt [hunk 2]
```diff
@@ -5,5 +5,5 @@
 #UO01948 UO01949 - IO27918 IO27919 - Fri Jul 10 13:23:34 UTC 2020 - Zowe 1.13.0
 #UO01951 UO01952 - IO27949 IO27950 - Wed Aug 12 14:01:06 UTC 2020 - Zowe 1.14.0
 #UO01953 UO01954 - IO27951 IO27952 - Fri Sep 11 12:48:24 UTC 2020 - Zowe 1.15.0
-UO01955 UO01956 - IO27953 IO27954 - 
+UO01955 UO01956 - IO27953 IO27954
 #UO01958 UO01959 - IO27974 IO27975 -
```
Comment (stevenhorsman) [inline smpe/bld/service/ptf-bucket.txt:9]: can you remove the trailing dash of the 1.17 line as I think it might confuse the automated promote.

**Review by stevenhorsman — CHANGES_REQUESTED** [review]
Code looks fine, but no PTF build or tests run.

**Comment by Roavill2** [comment]
Okay cheers - will kick off tests now

**Comment by Roavill2** [comment]
Link to passing tests:

smpe ptf tests - https://wash.zowe.org:8443/view/Zowe/job/zowe-install-test/view/change-requests/job/PR-1681/3/

conv tests - https://wash.zowe.org:8443/view/Zowe/job/zowe-install-test/view/change-requests/job/PR-1681/4/

**Review by stevenhorsman — APPROVED** [review]

**Pull request merged**